fearlessTests/Modules/Staking/StakingRewardPayouts/StakingRewardPayoutsTests.swift

```diff
@@ -1,12 +1,61 @@
 import XCTest
+import Cuckoo
+@testable import fearless
 
 class StakingRewardPayoutsTests: XCTestCase {
 
-    override func setUp() {
-        // Put setup code here. This method is called before the invocation of each test method in the class.
-    }
+    func testViewStateIsLoadingThenError() {
```

Comment:
We are not testing interactor here. I think we can use price provider stub (```SingleValueProviderFactoryStub.westedNominator().priceProvider```) and also we need to create PayoutService stub. Then we are make sure that services are called and some data returns back to presenter.